security/imx_ahab/sign-file.sh

```diff
@@ -131,5 +131,4 @@ echo "Process completed successfully and signed file is ${WORK_FILE}.signed"
 # Cleanup config / mod SPL
 rm ${CSF_TEMPLATE}.csf-config
 rm -f ${CSF_TEMPLATE}.csf-config~
-rm ${WORK_FILE}_csf.bin
```

Comment:
in the hab4 version of the cst file the output file was ```${WORK_FILE}_csf.bin``` and was just the signature that you then had to append to the original file. Does the ahab cst generate the full file directly?

Reply:
yes

Reply:
basically these commands are executed:

```
# working file used for signature
cp ${WORK_FILE} ${WORK_FILE}.mod

# generate the signatures, certificates, ... in the CSF binary
echo "Invoking CST to sign the binary"
${CST_BINARY} --o ${WORK_FILE}.signed --i ${CSF_TEMPLATE}.csf-config > $OUT

echo "Process completed successfully and signed file is ${WORK_FILE}.signed"
```